app/commands/v2/unpublish.rb

```diff
@@ -153,18 +155,13 @@ def find_unpublishable_content_item(content_id)
         filter.filter(locale: locale, state: allowed_states).last
       end
 
-      def supersede_previous_items(content_item)
-        items = ContentItemFilter.similar_to(
+      def lookup_previous_item(content_item)
+        ContentItemFilter.similar_to(
           content_item,
           state: %w(published unpublished),
           base_path: nil,
           user_version: nil,
-        )
-
-        items.each do |item|
-          next if item == content_item
-          State.supersede(item)
-        end
+        ).first
```

Comment:
Same as https://github.com/alphagov/publishing-api/pull/405/commits/ee5626ed7503e0615e6238de397dede2606a3463#r70237338 , it would be good to have an explicit error.